src/site/stages/build/drupal/load-saved-flags.js

```diff
@@ -27,7 +28,10 @@ function useFlags(rawFlags) {
         'inspect',
         'Symbol(Symbol.iterator)',
       ];
-      if (!ignoreList.includes(prop.toString())) {
+      if (
+        !ignoreList.includes(prop.toString()) &&
+        buildType !== ENVIRONMENTS.LOCALHOST // Don't require a cache to build locally.
```

Comment:
This one's a little different. My suggestion to add this comment was in the `shouldPullDrupal` function.
```suggestion
        buildType !== ENVIRONMENTS.LOCALHOST // Don't fail a localhost build for missing query flags
```

Reply:
Doh. Sorry I'll update.